tutorials/javaFxPart1.md

```diff
@@ -172,11 +173,12 @@ Congratulations! You have created your first GUI application!
    1. Try changing the `Label` to have a font of Arial at size 50.
 
 1. You’ve learnt that a `Stage` can be thought of as a window.
-   1. Can you have more than one `Stage` an application?
+   1. Can you have more than one `Stage` in an application?
    1. Try creating another stage and showing it! What happens?
 
 [:fas-arrow-up: **ToC**](javaFx.md) | <span class="badge rounded-pill bg-primary">**++What's next?++**</span> [:fas-arrow-right: JavaFX tutorial part 2 - **Creating a GUI for Duke**](javaFxPart2.md)
 
 --------------------------------------------------------------------------------
 **Authors:**
 * Initial Version: Jeffry Lum
+* Contributors: Zhang Lanyu
```

Comment:
We add name here only for significant contributions.

Reply:
Thank you! Fixed in new commit.